packages/components/src/custom-select-control-v2/custom-select.tsx

```diff
@@ -1,3 +1,9 @@
+/**
+ * External dependencies
+ */
+// eslint-disable-next-line no-restricted-imports
```

Comment:
🤔 Shouldn't we allow Ariakit imports within the components package? Sounds like it could be an exception to the ESLint rule.

Reply:
https://github.com/WordPress/gutenberg/pull/63123